### ISSUE TITLE
[Bug] [a11y] Accessibility Scanner suggests increasing the "x" button from Save to Collection page

### ISSUE THREAD

**Issue by softvision-miralobontiu** [mozilla-mobile/fenix#7672]
## Prerequisites
1. Make sure you have Google Accessibility Scanner installed.
2. Have a freshly installed build or a clean profile.

## Steps to reproduce
1. Open a few tabs.
2. From the "Open tabs" section tap on the "Save to collection" button.
3. Scan the collections tabs selection fragment.

### Expected behavior
No suggestions are made.

### Actual behavior
Accessibility Scanner suggests the "x" button should be larger: from 24dp x 24dp to 48dp x 48dp - as seen in the screenshot below.

### Device information
* Android device: Samsung Galaxy Tab S3 (Android 8)
* Fenix version: RC 3.0.2, Beta build 3.1.0-beta 3, Nightly build from 01/14

![Screenshot_20200114-115728_Accessibility Scanner](https://user-images.githubusercontent.com/40694158/72335841-48be1f80-36c8-11ea-8811-135ae479986a.jpg)

**Comment by Diana-Rus** [comment]
Hi, the issue is still reproducible on Nightly 4/1 (Build #20920607) with Samsung Galaxy S9 (Android 8)

**Comment by mcarare** [comment]
Fixed in https://github.com/mozilla-mobile/fenix/issues/9626. QA please confirm! TY!

**Comment by sflorean** [comment]
Tested on the Nightly build 6/3 with Google Pixel 3 (Android 10), Samsung Galaxy Note 10 (Android 10), and LG G7 FIT (Android 8), and no suggestions were made for the "x" button - select tabs menu.
